db/migrations/sqls/20200526142326-add-refresh-tokens-up.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE refresh_tokens (
+  id serial PRIMARY KEY,
+  value varchar(255) NOT NULL UNIQUE
```

Comment:
Assuming we are going to be searching this table by the token, do we need to index the value column? 

And also is a 255 length long enough for the tokens we will be generating ?

Reply:
255 was long enough in testing. Depending on how much we change the token though this will increase/decrease. 

We should probably add an idea. Will add another todo.

Reply:
Because this is the refreshToken, it should be smaller than our actual token, as it only needs a subset of fields.